pkg/glusterfs/client.go

```diff
@@ -15,20 +19,30 @@ import (
 	"k8s.io/client-go/rest"
 )
 
+type GlusterServerKind string
+
 const (
 	GD2DefaultInsecure = false
+
+	ServerKindUnknown GlusterServerKind = ""
+	ServerKindGD2     GlusterServerKind = "glusterd2"
```

Comment:
Style nit: When defining constants for a new type I think its a lot cleaner/clearer to start a new const (...) block, that only contains the constants for that type.